doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -17,19 +17,19 @@ MXNet\*
 Compile MXNet with nGraph
 --------------------------
 
-.. important:: These instructions pick up from where the :doc:`install`
-   installation instructions left off, so they presume that your system already
-   has the library installed at ``$HOME/ngraph_dist`` as the default location.
-   If the |nGl| code has not yet been installed to your system, please go back
-   and return here to finish compiling MXNet with ``libngraph``.
+.. important:: As of version |version|, these instructions presume that your 
+   system already has the Library installed to the default location, as outlined 
+   in our :doc:`install` documentation. If the |nGl| code has not yet been 
+   installed to your system, please go back and return here to finish compiling 
+   MXNet with ``libngraph``.
 
 
 #. Set the ``LD_LIBRARY_PATH`` path to the location where we built the nGraph 
    libraries:
 
    .. code-block:: bash
 
-      export LD_LIBRARY_PATH=$HOME/ngraph_dist/lib/
+      export LD_LIBRARY_PATH=path/to/ngraph_dist/lib/
```

Comment:
It is unfortunate if we have to tell the user to use `LD_LIBRARY_PATH`.  Is there a make flag that we can add to mxnet to tell it where ngraph lives? I also see on line 53 where we specify `NGRAPH_DIR`. Too many environment variables. This is not a problem with the documents but with our build process.

Reply:
@rkimballn1  The reason the documentation was originally structured this way is that setting the `LD_LIBRARY_PATH` is the first invocation where we let the "framework" be in charge of adding new stuff to whatever it needs to compile / be compiled.  Since this is a framework-specific setting (the first thing a framework needs to do is make the libraries available to the framework), we parsed this out and let this be a basic starting point for *any* framework, so that other framework developers not using TF or MXNet can be in charge of their own destinies.  Additional `LD_LIBRARY_PATH` config options are beyond the scope of nGraph since we already built nGraph in the previous install doc.